tests/TopDownProteomics.Tests/ProForma/ProFormaParserTests.cs

```diff
@@ -832,6 +838,7 @@ public void Ambiguity_PossiblePositions_4_4_2()
             var group = term.TagGroups.Single();
             Assert.AreEqual("g1", group.Name);
             Assert.AreEqual("Phospho", group.Value);
+            Assert.AreEqual(2, group.PreferredLocation);
```

Comment:
See comment above. I think you should add a test for 
`"EM[Oxidation]EVT[#g1]S[Phospho#g1]ES[#g1]PEK"` showing that this value is 1.

Reply:
Agreed, I'll add this. I believe the logic above was able to use the `currrentGroup.Members.Count` because that will effectively be the index at that point in the logic, but the unit test will directly test that.

Reply:
Unit test added and passing.